cmd/brew-test-bot.rb

```diff
@@ -446,6 +446,8 @@ def download
         diff_end_sha1 = diff_start_sha1 = current_sha1
       end
 
+      ohai "Testing commits #{diff_start_sha1} to #{diff_end_sha1}"
```

Comment:
should this go after the next statement, which sets `diff_start_sha1`?

Reply:
the macOS build prints `==> Testing commits  to 6cb92c0`

Reply:
We could put a second statement just before the `return if diff_start_sha1 == diff_end_sha1` line. As `diff_start_sha1` may be modified at least twice after this line. So we would get the initial value and the modified value.

Reply:
> the macOS build prints `==> Testing commits to 6cb92c0`

Well this could be because the start sha1 is not computed on Azure :\

Reply:
The question then is why is `diff_start_sha1` not computed on Azure?